packages/web3-atl-aspect/src/index.js

```diff
@@ -412,8 +414,6 @@ Aspect.prototype.upgrade = function (options, callback) {
         processedJPName.set(name, enumValue);
         if (enumValue !== undefined) {
```

Comment:
better not use things like `something !== undefined`, what if `something` is a `null` value? You can do `!!enumValue` which makes more sense.